config/samples/perf_v1alpha1_qperf.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: perf.kubestone.xridge.io/v1alpha1
+kind: Qperf
+metadata:
+  name: qperf-sample
+spec:
+  image:
+    name: xridge/qperf:0.4.11-r0
+    pullPolicy: IfNotPresent
+    # pullSecret: null
+
+  options: "--verbose --time 10"
+  tests:
+  - tcp_bw
+  - tcp_lat
+
+  serverConfiguration:
+    # nodeSelector: null
+    podLabels:
+      iperf-mode: server
```

Comment:
I'm not sure if needed, if so should be changed to qperf-mode

Reply:
The label is used for pod scheduling, renamed to qperf-mode. It was a copy-pasta from iperf3 :)